src/collections/_documentation/platforms/python/index.md

```diff
@@ -2,55 +2,398 @@
 title: Python
 ---
 
-{% include learn-sdk.md platform="python" %}
+## Integrating the SDK
 
-The [Sentry Python SDK](https://pypi.org/project/sentry-sdk) provides support
-for Python 2.7 and 3.4 or later.
+Sentry captures data by using an SDK within your application’s runtime. These are platform specific and allow Sentry to have a deep understanding of how your application works.
 
-This documentation goes over some Python specific things such as integrations to
-frameworks.
+Install our Python SDK using `pip`:
 
-## Integrations
+```bash
+$ pip install --upgrade sentry-sdk
+```
 
-*Integrations* extend the functionality of the SDK for some common frameworks and
-libraries. Similar to plugins, they extend the functionality of the Sentry
-SDK. A call to `sentry_sdk.init` configures integrations. Unless you set `default_integrations` to `False`, Sentry automatically adds any default integration not in the list below.
+### Connecting the SDK to Sentry
 
-### Web Frameworks
+After you’ve completed setting up a project in Sentry, Sentry will give you a value which we call a DSN or Data Source Name. It looks a lot like a standard URL, but it’s just a representation of the configuration required by the Sentry SDKs. It consists of a few pieces, including the protocol, public key, the server address, and the project identifier.
 
-* [Django]({% link _documentation/platforms/python/django.md %})
-* [Flask]({% link _documentation/platforms/python/flask.md %})
-* [Sanic]({% link _documentation/platforms/python/sanic.md %})
-* [Pyramid]({% link _documentation/platforms/python/pyramid.md %})
-* [AIOHTTP]({% link _documentation/platforms/python/aiohttp.md %})
-* [Tornado]({% link _documentation/platforms/python/tornado.md %})
-* [Bottle]({% link _documentation/platforms/python/bottle.md %})
-* [Falcon]({% link _documentation/platforms/python/falcon.md %})
-* [Generic WSGI]({% link _documentation/platforms/python/wsgi.md %})
+Import and initialize the Sentry SDK early in your application’s setup:
 
-### Task Queues
+```python
+import sentry_sdk
+sentry_sdk.init("https://<key>@sentry.io/<project>")
+```
 
-* [Celery]({% link _documentation/platforms/python/celery.md %})
-* [RQ (Redis Queue)]({% link _documentation/platforms/python/rq.md %})
+### Verifying Your Setup
 
-### Serverless
+Great! Now that you’ve completed setting up the SDK, maybe you want to quickly test out how Sentry works. Start by capturing an exception:
 
-* [AWS Lambda]({% link _documentation/platforms/python/aws_lambda.md %})
-* [Generic Serverless]({% link _documentation/platforms/python/serverless.md %})
+```python
+sentry_sdk.capture_exception(Exception("This is my fake error message"))
```

Comment:
It'd be good if we kept the same example error message throughout docs. Preferably something that reads well for non-English speakers as well. I bring this up because the JavaScript example has always bothered me.

e.g. "This is an example error"

Reply:
How about: "This is an example of an error message."